pylsp_ruff/plugin.py

```diff
@@ -405,7 +405,7 @@ def create_fix_all_code_action(
     document: Document,
     settings: PluginSettings,
 ) -> CodeAction:
-    title = "Ruff: Fix All"
+    title = "Ruff: Fix All (all safe fixes)"
     kind = CodeActionKind.SourceFixAll
 
     # No unsafe fixes for 'Fix all', see https://github.com/python-lsp/python-lsp-ruff/issues/55
```

Comment:
this is supposed settings.unsafe_fixes is default false and only to that,
or if can be switched to true, then the title perhaps should dyn to change.